src/hooks/forms/useUpdatePasswordForm.ts

```diff
@@ -0,0 +1,58 @@
+import { useAuthControllerUpdate } from '@baca/api/query/auth/auth'
+import { AuthUpdateDto } from '@baca/api/types'
+import { handleFormError, hapticImpact, showSuccessToast } from '@baca/utils'
+import { useMemo } from 'react'
+import { useForm } from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+
+export const useUpdatePasswordForm = () => {
+  const { t } = useTranslation()
+  const { mutate: updatePasswordMutation, isLoading } = useAuthControllerUpdate()
+
+  const defaultValues: AuthUpdateDto = useMemo(
```

Comment:
not needed inside hook, can be declared before hook without useMemo

Reply:
True, they're static values 🧠